packages/terra-tag/src/Tag.jsx

```diff
@@ -97,16 +116,19 @@ class Tag extends React.Component {
     const isClickable = href || onClick;
     const clickableComponent = href ? 'a' : 'button';
     const ComponentType = isClickable ? clickableComponent : 'span';
+    const tagRole = clickableComponent === 'a' ? 'link' : 'button';
```

Comment:
The role should be handled by the element being an anchor or button.  This logic does not seem necessary.

Reply:
Removed this lines. Thanks